org.lflang/src/org/lflang/generator/ts/TSReactionGenerator.kt

```diff
@@ -180,6 +207,21 @@ class TSReactionGenerator(
         }
     }
 
+    private fun generateMutationSignatureElementForPortEffect(effect: VarRef): String {
```

Comment:
Comments missing

Reply:
I wrote the comments for this function in https://github.com/lf-lang/lingua-franca/pull/1318/commits/dbec99308314d3e9cb63d9f47ace6c83530d1bfc